### PR TITLE
7102 use details/summary pairs to render FAQ

### PR DESCRIPTION
**Pull request by gmarinov**
7102

 - switches FAQ module markup to `<details><summary>` pairs (changes HTML markup)
 - example pids w/ FAQ: `b006qpgr`, `b006t1q9`

QA: 
worth checking that the following components also look and function correctly
- https://www.bbc.co.uk/programmes/b062jzdq mobile view, list of tracks
- https://www.bbc.co.uk/programmes/m0001vjc synopsis
- https://www.bbc.co.uk/programmes/b006q2x0/clips filter by

dev notes:

- I suggest that we do not polyfill this. IE11 renders it open by default. Other browsers actually understand the markup.
- adds a `target-ie` SASS mixin to use where we need to target IE>=9
- there's likely a bunch of CSS to remove from `_moreless.scss` - guidance welcome

#### modern browser output (`sandbox` branding)

Screenshot 2019-07-30 at 16 25 03

#### ie11 output (`test.` branding)

ie11-safe-ss

#### side by side (mobile-ish view: old ⟷ new )

Screenshot 2019-07-31 at 10 25 26

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: src/Ds2013/Presenters/Domain/ContentBlock/Faq/_faq.scss [hunk 1]
```diff
@@ -0,0 +1,48 @@
+$faq-details-indent: 18px;
+$faq-details-inner-indent: 26px;
+
+div.component[data-content-block-type='faq'] {
+    details {
+        margin: $faq-details-indent 0;
+
+        summary {
+            font-weight: bold;
+            font-size: 138.5%;
```
Comment (Tasiobg) [inline src/Ds2013/Presenters/Domain/ContentBlock/Faq/_faq.scss]: I never used `%` in font-size, usually `px` or `em`.
Is `%` or just personal preference?

Comment (gmarinov) [reply src/Ds2013/Presenters/Domain/ContentBlock/Faq/_faq.scss]: `px` is absoute. `em` is relative, `%` is relative to the `font-size` of the parent. But this here looks like it's not needed at all. Nice catch.

**Comment by BjornTwachtmann** [comment]
You should be able to remove all the references to "faq" from Ds2013/Css/Components/_moreless.scss . The other stuff will still be in use, see https://www.bbc.co.uk/programmes/b062jzdq (mobile view, list of tracks)  https://www.bbc.co.uk/programmes/m0001vjc (synopsis) https://www.bbc.co.uk/programmes/b006q2x0/clips (filter by). 

We should probably fix those at some point too... but not in this PR.

File: src/Ds2013/Presenters/Domain/ContentBlock/Faq/_faq.scss [hunk 2]
```diff
@@ -0,0 +1,46 @@
+$faq-details-indent: 18px;
+$faq-details-inner-indent: 26px;
+
+div.component[data-content-block-type='faq'] {
```
Comment (BjornTwachtmann) [inline src/Ds2013/Presenters/Domain/ContentBlock/Faq/_faq.scss]: Couldn't you just target details.faq-item below and remove this line?

Comment (gmarinov) [reply src/Ds2013/Presenters/Domain/ContentBlock/Faq/_faq.scss]: ah yes! thanks. done

**Pull request merged**